Use common AVX512 flagset to increase compatibility of bwa-mem2.avx512bw binary
across platforms supporting AVX512
see https://github.com/bwa-mem2/bwa-mem2/issues/236
author: Alex Domingo (Vrije Universiteit Brussel)
--- Makefile.orig	2024-04-29 14:52:21.634066000 +0200
+++ Makefile	2024-04-29 14:52:48.590282000 +0200
@@ -76,7 +76,7 @@
 	endif
 else ifeq ($(arch),avx512)
 	ifeq ($(CXX), icpc)
-		ARCH_FLAGS=-xCORE-AVX512
+		ARCH_FLAGS=-march=common-avx512 #-xCORE-AVX512
 	else	
 		ARCH_FLAGS=-mavx512bw
 	endif
